fpm-Dockerfile-block-2

```diff
@@ -33,7 +33,7 @@ RUN set -ex \
 		echo 'daemonize = no'; \
 		echo; \
 		echo '[www]'; \
-		echo 'listen = [::]:9000'; \
+		echo 'listen = 9000'; \
```

Comment:
Aw, the fix is 7.2+ only. :disappointed:

That's going to complicate the templating -- we've got to figure out a clean way to adjust this value based on whether we're PHP (5.6|7.0|7.1).

I think something like this is probably going to be our simplest fix:
```bash
# https://bugs.php.net/bug.php?id=74166
# https://github.com/php/php-src/commit/b63c45c69e3a97b05314c6f3328a0ff695e28d36
		case "$PHP_VERSION" in \
			5.*|7.0.*|7.1.*) echo 'listen = [::]:9000' ;; \
			*) echo 'listen = 9000' ;; \
		esac; \
```

Once that's updated, you should be able to simply run `./update.sh` (with no arguments) to update all the relevant `Dockerfile`s.

Reply:
Thank you for the review.  You are right that the fix is in >= 7.2.0 (not 7.2.1 as I have written).

I don't believe this needs to complicate the templating: `listen 'port'` is valid in all versions and before 7.2.0 `listen 9000` is interpreted as if the directive had been given as `listen [::]:9000`.  So the proposed conditional in the template is unnecessary.